auth/auth/auth.py

```diff
@@ -50,7 +51,6 @@
     PreviouslyDeletedUser,
     UnknownUser,
 )
-from .flow import get_flow_client
```

Comment:
in `hailtop.auth.login` I had to do something very similar of choosing which flow client to use but instead of having a cloud I had an `IdentityProvider` (I thought this distinction was valuable mentally if not in practice, maybe that's not true?). That led to me making a symmetric change, but I'll leave that change out for now.

Reply:
I can no longer find `get_flow_client`. Maybe I removed it days ago…

Reply:
flow.py is deleted, that's good enough for me